.github/pull_request_template.md

```diff
@@ -13,3 +13,6 @@ In order to reduce the number of notifications sent to the maintainers, please:
 - [ ] All filenames are in PascalCase.
 - [ ] All functions and variable names follow Java naming conventions.
 - [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
+- [ ] Ensure your code is properly formatted by using the command ```clang-format -i --style=file path/to/your/file.java```
+      If you do not have the correct version of clang-format (i.e. < 16.0), you can remove **InsertNewlineAtEOF: true** from
+      **java/.clang-format** and manually add a newline at the end of your file before pushing.
```

Comment:
```suggestion
- [ ] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`
```
I'd make it shorter and sound as a statement from the contributor, as other items above do.
Is this caveat about clang-format version important, or can we just omit it? It's too verbose to be in every PR description. If it's indeed helpful, we can make it as a comment below the checkbox so that it's only visible when editing the PR description.

Reply:
@siriak  Thanks for the review. I think the version info can be omitted. I have added the shorter statement as suggested by you.